### PR TITLE
Add range sensors and fuel level for ICE cars 

### PR DESCRIPTION
**Pull request by prvakt**
https://github.com/skodaconnect/homeassistant-myskoda/issues/239
https://github.com/skodaconnect/homeassistant-myskoda/issues/241

### CODE REVIEW

File: custom_components/myskoda/icons.json [hunk 1]
```diff
@@ -95,6 +95,15 @@
...
 				"default": "mdi:speedometer"
 			},
+            "combustion_range": {
+                "default": "mdi:car"
+            },
+            "electric_range": {
+                "default": "mdi:car-electric"
+            },
+            "fuel_level": {
+                "default": "mdi:fuel"
```
Comment (dvx76) [inline custom_components/myskoda/icons.json:105]: Optional: consider  `mdi:gauge`

File: custom_components/myskoda/icons.json [hunk 2]
```diff
@@ -95,6 +95,15 @@
 			"range": {
 				"default": "mdi:speedometer"
 			},
+            "combustion_range": {
+                "default": "mdi:car"
```
Comment (dvx76) [inline custom_components/myskoda/icons.json:99]: Optional: consider `mdi:gas-station`

File: custom_components/myskoda/icons.json [hunk 3]
```diff
@@ -95,6 +95,15 @@
...
+            },
+            "electric_range": {
+                "default": "mdi:car-electric"
```
Comment (dvx76) [inline custom_components/myskoda/icons.json:102]: Optional: consider `mdi:ev-station`

**Review by dvx76 — COMMENTED** [review]
Nice! Thanks for contributing. Some minor suggestions and I think some linter issues to address (just newlines if I saw correctly)

**Comment by WebSpider** [comment]
And a minor merge conflict, but thats solved by a rebase or merge-main.

File: custom_components/myskoda/icons.json (hunk 1 above)
Comment (prvakt) [reply custom_components/myskoda/icons.json:105]: I think mdi:fuel is more suitable

**Review by WebSpider — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**